feedback.md

```diff
@@ -0,0 +1,29 @@
+# INFO 343 Final Project
+
+August Carow, Katie Goulding, Amanda Shen, Matthew Farmer 
+
+<!-- Description of what we will build -->
+
+## Introduction:
+Our team is going to create a web application. The most basic function of the web app will be a timer that you can use to time the length of your shower. More importantly, the shower timer will estimate how much water you used and how much it costs. If you take an especially long shower, the app might prompt you with a depressing environmental fact. The app may also have functionality that allows you to see the difference between you and your friend’s water usage. The greater goal of this simple app is to encourage people to take shorter showers through a few different strategies, thus conserving water and energy.
+
+## Features:
+* Timer to know duration of shower
+* Determine volume of water used
+* Determine cost of that shower based on amount of water used and water cost in area (likely to initially be developed just for Seattle area, TBD).
+
+## Goal:
+Reduce water consumption to conserve water and raise awareness of the environmental crisis surrounding finite water resources.
+
+<!-- Who will build what -->
+
+## Who will do what:
```

Comment:
Not super clear on roles. Be sure you have clear assignments of who is doing what and by when. Be terrific, be specific.